File: rolling-update-kubernetes/src/test/scala/doc/akka/rollingupdate/kubernetes/PodDeletionCostCompileOnly.scala
/*
 * Copyright (C) 2017-2025 Lightbend Inc. <https://akka.io>
 */

package doc.akka.rollingupdate.kubernetes

import akka.actor.ActorSystem
import akka.rollingupdate.kubernetes.PodDeletionCost

object PodDeletionCostCompileOnly {

  val system = ActorSystem()

  //#start
  // Starting the pod deletion cost annotator
  PodDeletionCost(system).start()
  //#start

}

File: rolling-update-kubernetes/src/test/java/jdoc/akka/rollingupdate/kubernetes/PodDeletionCostCompileOnly.java
/*
 * Copyright (C) 2017-2025 Lightbend Inc. <https://akka.io>
 */

package jdoc.akka.rollingupdate.kubernetes;

import akka.actor.ActorSystem;
import akka.rollingupdate.kubernetes.PodDeletionCost;

public class PodDeletionCostCompileOnly {
    public static void bootstrap() {

        ActorSystem system = ActorSystem.create();

        //#start
        // Starting the pod deletion cost annotator
        PodDeletionCost.get(system).start();
        //#start
    }
}
